soc/nordic/nrf54l/soc.c

```diff
@@ -171,6 +172,17 @@ int nordicsemi_nrf54l_init(void)
 #if !defined(CONFIG_TRUSTED_EXECUTION_NONSECURE) || defined(__NRF_TFM__)
 	/* Currently not supported for non-secure */
 	SystemCoreClockUpdate();
+
+	/* Don't retain CRACEN RAMs during System OFF */
+	nrf_memconf_ramblock_ret_mask_enable_set(NRF_MEMCONF, 1, (1 << 2), false);
+	nrf_memconf_ramblock_ret_mask_enable_set(NRF_MEMCONF, 1, (1 << 3), false);
```

Comment:
Could those `(1 << X)` be defines to explain what they are?

Reply:
Could MEMCONF_POWER_RET_MEM2_Msk, MEMCONF_POWER_RET_MEM3_Msk and MEMCONF_POWER_CONTROL_MEM2_Msk and MEMCONF_POWER_CONTROL_MEM3_Msk be utilized for this?

Reply:
Please don't mix MDK defines with HAL functions, if you think those masks are needed we can add them to HAL.

Reply:
Yeah, I will not mix these up indeed. But I will write some  more description on the bits that I am setting now that I got a confirmation from the spec writers that it is ok to do so.